packages/kolibri-common/components/SyncSchedule/ManageSyncSchedule.vue

```diff
@@ -261,11 +276,49 @@
       },
       editButton(value) {
         if (value !== ' ') {
-          this.$router.push({ name: PageNames.EDIT_SYNC_SCHEDULE, params: { deviceId: value } });
+          this.$router.push({
+            name: SyncPageNames.EDIT_SYNC_SCHEDULE,
+            params: { deviceId: value, facilityId: this.facilityId },
+          });
         } else {
           return window.location.href;
         }
       },
+      ScheduleTime(time, timestamp) {
```

Comment:
Maybe change this to camelCase (scheduleTime), as we usually use that for Javascript/Vue methods.